db/Cassdb.js

```diff
@@ -0,0 +1,51 @@
+const cassandra = require('cassandra-driver');
+
+const client = new cassandra.Client({ contactPoints: ['127.0.0.1'], keyspace: 'languagefeatures' });
+
```

Comment:
creating a connection pool might make this run faster eg: const cassandra = require('cassandra-driver');
const distance = cassandra.types.distance;

const options = {
  contactPoints: ['127.0.0.1'],
  localDataCenter: 'datacenter1',
  pooling: {
    coreConnectionsPerHost: {
      [distance.local]: 2,
      [distance.remote]: 1
    } 
  }
};